src/styles/index.scss

```diff
@@ -1,3 +1,135 @@
-body {
+$font-family: 'Roboto', sans-serif;
+$main-color: #060b35;
+$second-color: #616070;
+
+body,
+p {
   margin: 0;
+  font-family: $font-family;
+  color: $main-color;
+}
+
+.product-card {
+  box-sizing: border-box;
+  display: flex;
+  width: 200px;
+  align-items: center;
+  flex-direction: column;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    padding-top: 32px;
+    padding-left: 19px;
+    padding-right: 19px;
+    margin-bottom: 40px;
+  }
+  &__name {
+    width: 166px;
+    height: 36px;
+    font-family: $font-family;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0;
+    margin-bottom: 4px;
+    color: $main-color;
```

Comment:
The `margin: 0;` declaration is redundant here since it's already declared in the `body, p` selector. It's better to remove unnecessary code to keep the stylesheet clean and maintainable.